dbms/src/TestUtils/ExecutorTestUtils.cpp

```diff
@@ -0,0 +1,130 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Common/FmtUtils.h>
+#include <Flash/Coprocessor/DAGQuerySource.h>
+#include <Interpreters/executeQuery.h>
+#include <TestUtils/ExecutorTestUtils.h>
+#include <TestUtils/executorSerializer.h>
+namespace DB::tests
+{
+DAGContext & ExecutorTest::getDAGContext()
+{
+    assert(dag_context_ptr != nullptr);
+    return *dag_context_ptr;
+}
+
+void ExecutorTest::initializeContext()
+{
+    dag_context_ptr = std::make_unique<DAGContext>(1024);
+    context = MockDAGRequestContext(TiFlashTestEnv::getContext());
+    dag_context_ptr->log = Logger::get("executorTest");
+}
+
+void ExecutorTest::SetUpTestCase()
+{
+    try
+    {
+        DB::registerFunctions();
+        DB::registerAggregateFunctions();
```

Comment:
Add registerWindowFunctions?

Reply:
The code here may have potential bug in unit test if we run function related test and MockExecutor test simultaneously(https://github.com/pingcap/tiflash/issues/5081), I wonder if I can fix it in another pr.